test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -590,6 +591,33 @@ class test_ucp_sockaddr : public ucp_test {
 
 unsigned test_ucp_sockaddr::m_err_count = 0;
 
+UCS_TEST_SKIP_COND_P(test_ucp_sockaddr, close_ep_force_before_err_cb,
+                     !(get_variant_value() & CONN_REQ_STREAM) ||
```

Comment:
why disable on stream?

Reply:
probably because of https://github.com/openucx/ucx/pull/5931/files#r565105815,
anyway all these conditions should be double checked before merge